ubuntu-advantage.1

```diff
@@ -1,157 +1,130 @@
+Each service status line has three columns:
 
-.SH FIPS (Canonical FIPS 140-2 certified modules)
-Install, configure, and enable FIPS 140-2 certified modules.
-.TP
-Enabling the FIPS PPA repository, installs the FIPS modules, configures
-the bootloader and enables fips on the system. After executing
-"ubuntu-advantage enable fips", the system MUST be rebooted to
-complete the enablement process. Failing to reboot will result in the system
-not being fips enabled.
+.BR "name" ":"
+name of the service
 
-The following FIPS certified modules will be installed and put in fips mode;
-openssh-server, openssh-client, strongswan, openssl, and the kernel
-cryptoapi.
+.BR "availability" ":"
+whether the contract entitles use of this service. Possible values are:
+none or entitled
 
-Disabling FIPS is not currently supported.
+.BR "status" ":"
+whether the service is activated on this machine.
+Possible values are: active, inactive, or n/a
 
-.SH FIPS (Canonical FIPS 140-2 certified modules with updates)
-Install, configure, and enable FIPS 140-2 certified modules with updates
-.TP
-.B
-Enabling the FIPS-updates entitlement with "ua enable fips-updates" will take
-the system out of FIPS compliance as the updated modules are not FIPS
-certified. The option enables the FIPS-UPDATES PPA repository and installs the
-updated FIPS modules. If the system is installing FIPS modules for the first
-time, it configures FIPS on the system. After successfully executing the
-ubuntu-advantage to update FIPS modules, the system MUST be rebooted if FIPS
-kernel was upgraded in the upgrade process. Failing to reboot will result
-in the system not running the updated FIPS kernel.
-
-The following FIPS modules will be updated and put in FIPS mode -
-openssh-server, openssh-client, strongswan, openssl, and the kernel
-cryptoapi.
-
-.SH Livepatch (Canonical Livepatch Service)
-Managed live kernel patching. For more information, visit
-https://www.ubuntu.com/server/livepatch
 .TP
-Enable the Livepatch service on this machine to allow critical kernel patches
-without requiring a reboot.
+.B version
+Show version of the Ubuntu Advantage package.
 
-.SH EXIT STATUS
-.TP
-.B
-0
-Command succeded
-.TP
-.B
-1
-Invalid command or option
-.TP
-.B
-2
-Current user is not root
-.TP
-.B
-3
-Invalid or missing token when enabling a service
 .TP
-.B
-4
-The requested service is not supported on the current Ubuntu release
+.B detach
+Remove the Ubuntu Advantage support contract from this machine. This
+also disables all enabled services that can be.
+
+
+.SH SERVICES
 .TP
-.B
-5
-Current kernel is too old to support Snaps (required for the Livepatch service)
+.B "Common Criteria EAL2 Provisioning (cc)"
+Enables and install the common criteria artifacts.
+
+The artifacts include a configure script, a tarball with additional
+packages, and post install scripts. The artifacts will be installed in
+/usr/lib/common-criteria directory. The evaluated configuration guide
```

Comment:
I'm not sure what evaluated means in "evaluated configuration guide"; is it a term of art in Common Criteria?

Reply:
I copied this from the existing one and started to re-word, but seems I need to continue to re-word